localize-templates.py

```diff
@@ -0,0 +1,97 @@
+#! /usr/bin/env python3
+'''
+This script aims to replace unlocalised templates in localised pages with the localised templates.
+'''
+import os
+import copy
+from ws.client import API
+import ws.ArchWiki.lang as lang
+import mwparserfromhell as mw
+
+
+localised_lang_name: str = "简体中文" # not required, can omit val; simplified Chinese (zh-Hans)
+templates: list[str] = ["Translateme"]
+api_url = "https://wiki.archlinux.org/api.php"
+index_url = "https://wiki.arclinux.org/index.php"
+cookie_path = os.path.expanduser("~/.cache/ArchWiki.cookie")
+session = API.make_session(ssl_verify=True,
+                           cookie_file=cookie_path)
+
+
+def edit(api: API, timestamp: str, text: str, page, template: str, language: str):
+    code = mw.parse(text)
+    for curTemplate in code.filter_templates():
+        if curTemplate.name.matches(template):
+            curTemplate.name = f"{template} ({language})"
+    text = str(code)
+    api.edit(pageid=page['pageid'], title=page['title'], basetimestamp=timestamp, text=new, summary="localize templates", bot="")
```

Comment:
Can you use the [edit_interactive](http://lahwaacz.github.io/wiki-scripts/ws/ws.interactive.html#ws.interactive.edit_interactive) wrapper here? It would be useful for testing...